main/autogen/autogen.cc

```diff
@@ -663,15 +664,116 @@ string ParsedFile::toMsgpack(core::Context ctx, int version) {
     return write.pack(ctx, *this);
 }
 
-void ParsedFile::classlist(core::Context ctx, vector<string> &out) {
-    auto nameToString = [&](const auto &nm) -> string { return nm.data(ctx)->show(ctx); };
+vector<string> ParsedFile::listAllClasses(const core::Context ctx) {
+    vector<string> out;
+
     for (auto &def : defs) {
         if (def.type != Definition::Class) {
             continue;
         }
         auto names = showFullName(ctx, def.id);
-        out.emplace_back(fmt::format("{}", fmt::map_join(names, "::", nameToString)));
+        out.emplace_back(fmt::format("{}", fmt::map_join(names, "::", [&ctx](const core::NameRef &nm) -> string {
+                                         return nm.data(ctx)->show(ctx);
+                                     })));
+    }
+
+    return out;
+}
+
+optional<AutogenSubclassMap> ParsedFile::listAllSubclasses(const core::Context ctx,
```

Comment:
`const` is redundant in front of `core::Context`. It's fine to keep it, but it actually doesn't do anything as all fields of `core::Context` are `const` anyway

Reply:
Done in https://github.com/sorbet/sorbet/pull/1040/commits/1b11b80b4a6a5905497c6040a2b7c9fa22faf7a0